subprojects/assemble/build-trace/src/main/kotlin/com/avito/android/build_trace/BuildTracePlugin.kt

```diff
@@ -21,7 +21,7 @@ public abstract class BuildTracePlugin : Plugin<Project> {
     }
 
     private fun registerListeners(project: Project, extension: BuildTraceExtension) {
-        val isEnabled = extension.enabled.getOrElse(false)
+        val isEnabled = extension.enabled.getOrElse(true)
```

Comment:
The most common case is to apply and use plugins.
It's inconvenient to have it disabled by default.